package.json

```diff
@@ -9,7 +9,18 @@
     "test": "lerna run test --stream --concurrency 1",
     "lint": "eslint **/*.js -c .eslintrc.json && markdownlint **/*.md --ignore **/node_modules/**",
     "screenshots": "lerna run screenshots --stream --concurrency 1",
-    "start": "lerna run bundle && npm start --prefix ./packages/d3fc"
+    "start": "lerna run bundle && npm start --prefix ./packages/d3fc",
+    "commitlint": "commitlint --from HEAD~1 to HEAD --verbose"
+  },
+  "husky": {
+    "hooks": {
+      "prepare-commit-msg": [
+        "echo +----------------------------------------------+ &&",
```

Comment:
This is windows specific ...

```
husky > prepare-commit-msg (node v10.7.0)
/bin/sh: -c: line 0: syntax error near unexpected token `&&'
```

Reply:
I have moved this into a js file inside `/scripts` to avoid this issue and to declutter the `package.json` file